faker/compat.py

```diff
@@ -1,6 +1,6 @@
 # Python 2/3 compat
 import sys
-if sys.version < '3':
+if sys.version_info[0] < 3:
```

Comment:
Thanks for this... the only thing I'd suggest is `sys.version_info.major` because it is even more obvious